context.go

```diff
@@ -59,6 +59,11 @@ type IncomingCall interface {
 	// If the caller is an ephemeral peer, then the HostPort cannot be used to make new
 	// connections to the caller.
 	RemotePeer() PeerInfo
+
+	// CallOptions returns the call options set for the incoming call. It can be useful
+	// if you are forwarding a request and wish to retain the CallerName(), which is not
+	// possible to set manually.
+	CallOptions() *CallOptions
```

Comment:
They're basically in date added order, not very useful, but I'd like to preserve history for when they were added, so can leave as is.

For the name, I think it's a little generic, but it's on the `IncomingCall` interface, so I think it makes sense (reads to me like incoming call options, which is what it is).